sdk/iothub/azure-iot-deviceprovisioningservice/README.md

```diff
@@ -0,0 +1,215 @@
+
+# Create client using endpoint and credential
+client = ProvisioningServiceClient(endpoint="https://my-dps.azure-device-provisioning.net/", credential=credential)
+```
+
+#### Using a DPS connection string:
+Depending on your use case and authorization method, you may prefer to initialize a client instance with a DPS
+connection string instead of providing the endpoint URL and credential separately. To do this, pass the DPS
+connection string to the client's `from_connection_string` class method:
+
+```python
+from azure.iot.deviceprovisioningservice import ProvisioningServiceClient
+
+connection_string = "Hostname=https;SharedAccessKeyName=xxxx;SharedAccessKey=xxxx"
+client = ProvisioningServiceClient.from_connection_string(connection_string=connection_string)
+```
+
+### Async Clients 
+This library includes a complete async API supported on Python 3.5+. To use it, you must
+first install an async transport, such as [aiohttp](https://pypi.org/project/aiohttp/).
+See
+[azure-core documentation](https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/core/azure-core/CLIENT_LIBRARY_DEVELOPER.md#transport)
+for more information.
+
+## Key concepts
+The following operation groups comprise the Service data plane layer:
+1. [Individual enrollments](https://learn.microsoft.com/azure/iot-dps/concepts-service#individual-enrollment)
+2. [Enrollment groups](https://learn.microsoft.com/azure/iot-dps/concepts-service#enrollment-group)
+3. [Device registration](https://learn.microsoft.com/azure/iot-dps/concepts-service#registration-record)
+
+The Device Provisioning Service client library for Python allows you to interact with each of these components through different operation namespaces on the ProvisioningServiceClient.
+
+## Examples
+The following sections provide several code snippets covering some of the most common DPS service, including:
+
+* [Create an individual device enrollment](#create-an-individual-device-enrollment "Create an individual device enrollment")
+* [Create an enrollment group](#create-an-enrollment-group "Create an enrollment group")
+* [Check device registration status](#check-device-registration-status "Check device registration status")
+
+### Create an individual device enrollment
+Create an individual enrollment to provision an individual device using symmetric key attestation.
+```python
+from azure.iot.deviceprovisioningservice import ProvisioningServiceClient
+
+# Initialize client
+client = ProvisioningServiceClient.from_connection_string(connection_string="<connection_string>")
+
+# Load certificate contents
+
+# Create a symmetric key individual enrollment
+client.individual_enrollments.create_or_update(
+    id="<enrollment_id>",
+    enrollment = {
+        "registrationId": "<enrollment_id>"
+        "attestation": {
+            "type": "symmetricKey",
+        },
+    }
+)
+```
+
+### Create an enrollment group
+Create an x509 enrollment group to provision an individual device using x509 attestation.
```

Comment:
"Create an x509 enrollment group to provision **one or more** devices using x509 attestation." ?